src/notifications/android_plugin.md

```diff
@@ -0,0 +1,128 @@
+    // using the `ExampleForegroundService` handler
+    override fun startForegroundService(message: Message) {
+        val intent = Intent(applicationContext, ExampleForegroundService::class.java)
+        intent.putExtra(Constants.EXTRA_REMOTE_MESSAGE, message)
+        ContextCompat.startForegroundService(applicationContext, intent)
+    }
+}
+```
+
+Now lets add the foreground service implementation. This should implement the notification plugin `ForegroundService` class, which handles the incoming notification intent and processes the event. To properly implement this, your class needs to override the `onCreate`, `getConnectRequest` and `getServiceConfig` functions. The `getConnectRequest` function is called by the `ForegroundService` to get a BreezSDK `ConnectRequest` which contains the data necessary to connect to the SDK. This data includes the `Config` with it's `workingDir` and the mnemonic.
+
+<div class="warning">
+<h4>Developer note</h4>
+In Android reading from secured storage can vary a lot depending if it is a Kotlin, Flutter or React Native based application and the dependencies used to write to the secured storage. Consult the dependency used to write to the secured storage on how to read data back from them.
+</div>
+
+```kotlin
+package com.example.application
+
+import breez_sdk_liquid.ConnectRequest
+import breez_sdk_liquid.defaultConfig
+import breez_sdk_liquid.LiquidNetwork
+import breez_sdk_liquid_notification.ForegroundService
+import breez_sdk_liquid_notification.NotificationHelper.Companion.registerNotificationChannels
+
+class ExampleForegroundService : ForegroundService() {
+    companion object {
+        private const val TAG = "ForegroundService"
+        private const val ACCOUNT_MNEMONIC = "BREEZ_SDK_LIQUID_SEED_MNEMONIC"
+    }
+
+    // Override the `onCreate` function
+    override fun onCreate() {
+        super.onCreate()
+        // Register the default notification channels
+        registerNotificationChannels(applicationContext)
+    }
+
+    // Override the `getConnectRequest` function
+    override fun getConnectRequest(): ConnectRequest? {
+        val config = defaultConfig(LiquidNetwork.MAINNET)
+
+        // Set the workingDir as the same directory as the main application
+        config.workingDir = "${applicationContext.filesDir}/breezSdkLiquid"
+
+        // Get the mnemonic from secured storage using an implementation of
+        // `readSecuredValue` depending on how data is written to secured storage.
+        // See Developer Note
+        return readSecuredValue(applicationContext, ACCOUNT_MNEMONIC)
+            ?.let { mnemonic ->
+                ConnectRequest(config, mnemonic)
+            }
+    }
+}
+```
+
+## Reference implementation
+For a complete reference, see how we implemented it in misty-breez wallet: [BreezFcmService.kt](https://github.com/breez/misty-breez/blob/main/android/app/src/main/kotlin/com/cBreez/client/BreezFcmService.kt).
```

Comment:
This gives a 404 error.

Reply:
These reference implementations I'm working on adding them now, so they're be not available for a bit. But I do see an error in this link, will correct it